tools/addlicense/variables.go

```diff
@@ -0,0 +1,45 @@
+package main
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+	"time"
+)
+
+var licenseText = fmt.Sprintf(`Copyright %d Flant JSC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+`, time.Now().Year())
+
+var goLicense = "/*" + licenseText + " */\n\n"
```

Comment:
```suggestion
var goLicense = "/*\n" + licenseText + "*/\n\n"
```

Just for better readability.

Before:
```
/*Copyright...
limitations under the License.
 */

```


After:
```
/*
Copyright...
limitations under the License.
*/